README.md

```diff
@@ -108,6 +108,12 @@ Usage: <converter.py> -i INPUT_FILE(S) -o OUTPUT_FILE -m nesdis -k maskout -t 0.
 ```
 For method option (-m) of bias and uncertainty calculation (default/nesdis), deafult means to set bias and uncertainty as 0.0 and nesdis means to use NESDIS bias and uncertainty calculation method. For maskout option (-k) default/maskout, default means to keep all missing values and maskout means to not write out missing values. For thinning option, the value should be within 0.0 and 1.0 depending how much data will be thinned, and 0.0 means without any thining.
 
+The AOD converter normally produces a single IODA output file, but it can also produce two additional output files, one containing only smoke-affected obs and one containing only dust-affected obs. This functionality uses flagging in the Aerosol Data Product (ADP) files created as companion datasets for the native AOD550 observation files. To activate the smoke and dust processing, use the --adp_mask flag followed by the ADP file(s) that correspond to the AOD file(s) specified with -i.
+
+```
+Usage (with smoke and dust processing): <converter.py> -i AOD_INPUT_FILE(S) --adp_mask ADP_INPUT_FILE(S) --adp_conf_lvl 'medhigh' -o OUTPUT_FILE -m nesdis -k maskout -t 0.0
```

Comment:
```suggestion
Usage (with smoke and dust processing): <converter.py> -i AOD_INPUT_FILE(S) --adp_mask ADP_INPUT_FILE(S) --adp_conf_lvl 'medhigh' -o OUTPUT_FILE -m nesdis -k maskout -n 0.0
```

think the `-t` is a typo and it should be `-n` for thinning?  I've only used `-n`... and as we work on this will change these flags to be more descriptive and should be able to condense

the dual capability can likely be made clearer in the help message from the parser too, will look into that

Reply:
Yes, your change is correct. I like the approach @jeromebarre suggested in our meeting earlier this week of using more long form arguments (e.g. --method instead of -m). The -i and -o flags are in common enough use for input and ouput files that those are easy to understand as they are, but it would definitely be more straightforward for all of the other arguments to be long form. If nothing else it will keep me from forgetting that -t isn't the correct flag for thinning.